spec/lib/fake_names_spec.rb

```diff
@@ -0,0 +1,43 @@
+require 'rails_helper'
+require 'spec_helper'
+
+describe "fake names" do
+  let(:dummy_class) {
+    Class.new {
+      include FakeNames
+      def initialize(name)
+        @name = name
+      end
+      attr_reader :name
+    }
+  }
+  subject(:c) { dummy_class.new(name) }
+
+  context 'with valid name' do
+    let(:name) {"real model"}
+    it "should have a real name" do
+      expect(c.has_real_name?).to be true
+    end
+  end
```

Comment:
I'd want you to meta test this; like have an array of 'invalid' names and an array of 'valid' names, and go through them all.